static/src/js/__tests__/App.test.js

```diff
@@ -36,86 +216,112 @@ function setup() {
 
   const props = {}
 
-  const enzymeWrapper = shallow(<App {...props} />)
+  render(<App {...props} />)
 
   return {
-    enzymeWrapper,
     props
   }
 }
 
-beforeEach(() => {
-  jest.clearAllMocks()
-  jest.spyOn(AppConfig, 'getApplicationConfig').mockImplementation(() => ({ env: 'dev' }))
-  jest.spyOn(AppConfig, 'getEnvironmentConfig').mockImplementation(() => ({ edscHost: 'https://search.earthdata.nasa.gov' }))
-})
-
 describe('App component', () => {
-  test('sets the correct default title', () => {
-    const { enzymeWrapper } = setup()
-    const helmet = enzymeWrapper.find(Helmet)
-    expect(helmet.props().defaultTitle).toEqual('Earthdata Search')
+  test('sets the correct default title', async () => {
+    setup()
+    await waitFor(() => expect(document.title).toEqual('Earthdata Search'))
   })
 
-  test('sets the correct title template', () => {
-    const { enzymeWrapper } = setup()
-    const helmet = enzymeWrapper.find(Helmet)
-    expect(helmet.props().titleTemplate).toEqual('[DEV] %s | Earthdata Search')
+  // TODO this is just a prop being passed in Helmet its not rendered on DOM so I think that we should remove it for RTL test
```

Comment:
Remove or resolve `TODO`

Reply:
Ah thats right. I added this since I wasn't able to migrate this test. Since our overall test coverage has increased and the meta element value is being checked in a different test I am suggesting to delete it